tutorial/content/build-system/_index.md

```diff
@@ -49,12 +49,24 @@ There are multiple Software components on this repo, such as feeders, simulators
 If it is forgotten to be mentiond in the README, one way of determining whether a separate build is needed or not is to check the package statement in the source code.
 If it says "package main" it is a separate executable and shall then be built and run as described above.
 
-### Loggging
+### Logging
 Logging can be command line configured at startup.
 * logging level can be set to either of [trace, debug, info, warn, error, fatal, panic].
 * logging output destination. It can either be written to file, or directed to standard output.
 The levels currently used are mainly info, warn, error. Info is appropriate during testing and debugging, while error is appropriate when performance is important.
 
+### Transport protocols
+Besides the transport protocols
+* HTTP
+* Websocke
+* MQTT
+that the specification lists, this server also supports
+* gRPC
+They are all except MQTT activated by the server per default.
```

Comment:
Lazyness is probably the main reason. I have many bad behaviors.

Reply:
I think its possible to go too far the other way. I don't think you need to say send 20 PRs if its a small team who know ever other.

Instead, for something like this I think it can be enough to simply do say the doc clean up in its own commit. It can be included in a larger PR if its easier. Which PR its included in does not matter so much once merged as then as the commits seperate the work.

Reply:
I agree that separate commits is probably an acceptable solution here. I will try to follow that in coming PRs.